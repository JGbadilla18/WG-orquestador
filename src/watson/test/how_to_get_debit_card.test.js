const testWatson = require('./testWatson');

watsonText = 'Como adquirir una tarjeta de debito.'

it(watsonText, () => {
  expect.assertions(1);
  return testWatson(watsonText).then(data => expect(data).toMatchSnapshot());
});
