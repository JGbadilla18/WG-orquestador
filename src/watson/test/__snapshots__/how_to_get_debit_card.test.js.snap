// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Como adquirir una tarjeta de debito. 1`] = `
Object {
  "actions": Array [],
  "components": Array [
    Object {
      "buttonType": "web_url",
      "buttons": Array [
        Object {
          "title": "Conoce más aquí",
          "url": " https://blog.corporacionbi.com/tarjeta-de-debito",
        },
      ],
      "response_index": 0,
      "response_type": "button",
      "text": " Para obtener una tarjeta de débito debes tener una cuenta monetaria, puedes solicitarla acercándote a Servicio al Cliente en cualquiera de nuestras Agencias Bi.",
    },
    Object {
      "response_type": "pause",
      "time": 10000,
      "typing": true,
    },
  ],
}
`;
